lib/dslash_domain_wall_5d.in.cu

```diff
@@ -0,0 +1,20 @@
+#include <dslash_domain_wall_5d.hpp>
+
+namespace quda
+{
+
+  constexpr QudaPrecision precision = QUDA_@QUDA_DSLASH_PREC@_PRECISION;
+  constexpr int nColor = @QUDA_DSLASH_NCOLOR@;
+  constexpr int reconI = @QUDA_DSLASH_RECONI@;
+
+  typedef @QUDA_DSLASH_DDARG@ DDArg;
```

Comment:
These `typedef` here (and in similar files) should be replaced with `using` since that's more in line with the QUDA style, i.e., `using DDArg = @QUDA_DSLASH_DDARG@;`